sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -49,6 +49,8 @@ class SessionCatalog(
     hadoopConf: Configuration) extends Logging {
   import CatalogTypes.TablePartitionSpec
 
+  val DEFAULT_DATABASE = "default"
```

Comment:
Anyway, I'll make that first.
It's easy to revert one commit.
And, if possible, I prefer to have `object SessionCatalog` for the future